release/gcb/cloud_builder.sh

```diff
@@ -75,6 +75,33 @@ add_license_to_tar_images "${REL_DOCKER_HUB}" "${CB_VERSION}" "${OUTPUT_PATH}"
 # log where git thinks the build might be dirty
 git status
 
+#Handle CNI artifacts.
+pushd ../cni
+CNI_OUT=$(make DEBUG=0 where-is-out)
+# CNI version strategy is to have CNI run lock step with Istio i.e. CB_VERSION
+MAKE_TARGETS=(build)
```

Comment:
I wonder why you need build target variables here and below?

Reply:
I guess technically I don't but was keeping this more consistent with the above code.  I can change this.